src/pybind/mgr/dashboard/frontend/src/app/shared/components/card-row/card-row.component.html

```diff
@@ -165,12 +168,22 @@
   </span>
 </ng-template>
 
-<ng-template #simplifiedSummary>
-  <span *ngIf="!dropdownTotalError else showErrorNum">
-    {{ data }}
-    <i class="text-success"
-       [ngClass]="[icons.success]"></i>
-  </span>
+<ng-template #hostSummary>
+  <ng-container *ngIf="!dropdownTotalError else showErrorNum">
+    <span>
+      {{ data.up }}
+      <i class="text-success"
+         *ngIf="data.up || data.up === 0"
+         [ngClass]="[icons.success]"></i>
+    </span>
+    <span *ngIf="data.down"
+          class="ms-2">
+         {{ data.down }}
+      <i class="text-danger"
+         [ngClass]="[icons.danger]">
+      </i>
```

Comment:
We should have  entry for maintenance as well.

Reply:
Added maintenance as a warning status